app/src/app/views/stix/mitigation/mitigation-list/mitigation-list.component.html

```diff
@@ -1,6 +1,22 @@
 <div class="list-page">
     <div class="list-page-header">
-        <h1>mitigations</h1>
+        <h1 class="with-help-icon">
+            mitigations <mat-icon [popover]="PopoverTag" class="help-icon" aria-label="help">help_outline</mat-icon>
+        </h1>
+
+        <popover-content #PopoverTag 
+                popoverPlacement="bottom"
+                [animation]="true" 
+                [closeOnClickOutside]="true"
+                 >
+            <div class="popover-description">
+                <p>
+                    Mitigations represent security concepts and classes of technologies that can be used to prevent a technique or sub-technique from being successfully executed. They support the standard set of fields and must be assigned to a domain.
+                    <br><br>
+                    A special mitigation published within the Enterprise domain, "Do Not Mitigate," should be used to mark any techniques which should not be mitigated.
+                </p>
```

Comment:
- Again, use two `<p>` tags instead of `<br>`s. 
- Not sure if the 2nd paragraph is useful here. It's interesting, but people in this UI aren't currently working on the technique-mitigation relationships. So I think we may want to remove the 2nd paragraph.